### ISSUE TITLE
Menu: keyboard focus issue with menu items (v5.6.0-0)

### ISSUE THREAD

**Issue by ushrivastava123** [eBay/ebayui-core#1172]
@imcburnie
when I navigate through ebay-menu-button-item via keyboard, the focus rings appear correctly
however when I click through individual item, focus rings are missing.
Markup:
```
 <div id="menu_basic" class="grid__cell grid__cell--4of16 grid__cell--4of16-lg grid__cell--4of8-sm" draggable="true" ondragstart="drag(event)">
      <h3>Basic</h3>
      <ebay-menu-button text="eBay Menu" on-menu-button-select(console.log.bind(console, "menu-sel evectent is triggered")) on-menu-button-collapse(console.log.bind(console, "menu-collapse evectent is triggered"))>
        <ebay-menu-button-item>item 1</ebay-menu-button-item>
        <ebay-menu-button-item>item 2</ebay-menu-button-item>
        <ebay-menu-button-item>item 3</ebay-menu-button-item>
      </ebay-menu-button>
    </div>
```
Buggy (10.6.0-0)
![image](https://user-images.githubusercontent.com/57548118/87357849-94d40a80-c519-11ea-97dc-9e89887ade07.png)

Non-buggy (10.5.0-0)
![image](https://user-images.githubusercontent.com/57548118/87357917-af0de880-c519-11ea-865e-f1de4def77be.png)

**Comment by ianmcburnie** [comment]
So I can verify that the issue is not present on the skin side in 10.6.0 branch, so I have moved this ticket to coreui for investigation.

**Comment by ianmcburnie** [comment]
Related to the following change causing an issue with rovingtabindex logic:

https://github.com/eBay/ebayui-core/pull/1160/files#diff-ea16c9f6e2d541d4df9ad58a4be943dbR70

I'm going to revert that line for now. It just means that any disabled menu item will still be able to receive keyboard focus, but will continue to be greyed out.  Longer term fix would involve some changes to makeup-rovingtabindex.

**Comment by ianmcburnie** [comment]
Fixed.
